checker/raw_result.go

```diff
@@ -84,3 +54,60 @@ type Tool struct {
 	Desc          string
 	ConfigFiles   []File
 }
+
+// SecurityPolicyData contains the raw results
```

Comment:
A lot of this code was added in previous PRs and somewhat disappeared too. Not really part of this PR.